src/change_import_style.ts

```diff
@@ -30,7 +30,7 @@ function replaceRecursiveFilePaths(args: any) {
 }
 
 function removeJsExtensionsFromRequires(contents: string) {
-  return contents.replace(/(require\(.*).js/g, (_, captureGroup: string) => {
+  return contents.replace(/(require\(.*)\.js/g, (_, captureGroup: string) => {
```

Comment:
What does this fix?

Reply:
There is a problem with using the `use_grpc_json=True` option. This replace changes the import from `require("@grpc/grpc-js")` to `require("@grpc/grpc")`. This change escapes the dot so that it matches the `.js` instead of `js`.